template/en/default/account/auth/login-small.html.tmpl

```diff
@@ -53,12 +53,15 @@
            name="Bugzilla_login"
            title="Login"
            placeholder="email address"
+           type="email"
```

Comment:
If you wouldn't mind claiming https://bugzilla.mozilla.org/show_bug.cgi?id=1141961 then?